### PR TITLE
Gender filters

### PR DESCRIPTION
**Pull request by tammar96**
Gender filters

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -113,8 +113,8 @@ def init():
             if len(room) == 3 and (dorm.block().startswith("a") or dorm.block().startswith("d")):
                 room = "0" + room
 
-            for c in ["a", "b", "c"]:
-                domain = str(dorm.block()) + "-" + room + c + ".kn.vutbr.cz"
+            for socket in ["a", "b", "c"]:
```
Comment (europ) [inline main.py]: Could use any other name for this? E.g. `access_point` or something like this.

File: main.py [hunk 2]
```diff
@@ -126,6 +126,10 @@ def init():
             print_footer()
 
         for person in names:
+            if dorm.boysOnly() and person.female():
```
Comment (europ) [inline main.py]: I think we may change the `female` class method to `isFemale` for better readability in this case. Could you, please, chage it too?

**Comment by xbolva00** [comment]
README needs to be updated too

File: main.py (hunk 1 above)
Comment (tammar96) [reply main.py]: No it is socket in the wall so socket is the appropriate name.

File: main.py (hunk 2 above)
Comment (tammar96) [reply main.py]: yup

**Comment by davidbolvansky** [comment]
Wow guys, epic release !

**Comment by tammar96** [comment]
@europ Review it pls; I want to merge it after almost 2 years.....

**Comment by tammar96** [comment]
@europ  it's rebased against master.

**Pull request closed without merging**